### PR TITLE
feat: support country name

### PR DESCRIPTION
**Pull request by dadiorchen**
## Description

Need to add a new API env variable to set the query API URL. 
Assuming that the capture API (for the capture pic on the left) would return with `lat` and `lon` properties.

image

**Issue(s) addressed**

- Resolves #_[insert ID of the issue addressed]_

**What kind of change(s) does this PR introduce?**

- [ ] Enhancement
- [ ] Bug fix
- [ ] Refactor

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Issue

**What is the current behavior?**

**What is the new behavior?**

## Breaking change

**Does this PR introduce a breaking change?**

### CODE REVIEW

File: .env.test [hunk 1]
```diff
@@ -1,2 +1,2 @@
 REACT_APP_WEBMAP_DOMAIN=http://test.treetracker.org
-
+REACT_APP_QUERY_API_ROOT=https://dev-k8s.treetracker.org/query
```
Comment (dadiorchen) [inline .env.test:2]: Temporarily use dev env for the time being, I will deploy the query API to test env in the near future.

File: src/components/CaptureMatching/CaptureImage.js [hunk 2]
```diff
@@ -10,6 +10,40 @@ import SkipNextIcon from '@material-ui/icons/SkipNext';
...
+  React.useEffect(() => {
+    setContent('loading...');
+    fetch(
+      `${process.env.REACT_APP_QUERY_API_ROOT}/countries?lat=${lat}&lon=${lon}`
+    )
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        } else if (res.status === 404) {
+          setContent('Can not find country at this place');
```
Comment (nmcharlton) [inline src/components/CaptureMatching/CaptureImage.js:28]: Maybe we just fall back to displaying raw lat, lon in this case, or nothing at all.
I don't think this message is very useful to the user.

Comment (dadiorchen) [reply src/components/CaptureMatching/CaptureImage.js:28]: But if we can't find country for the coordinates, usually it means something wrong, so we'd better let people know that, and if we just simple show lat/lon here, it's a bit confusion for people, why sometimes it's country name and sometimes it's not. I can make this message more clear: `cannot find country at [lat, lon]`

Comment (nmcharlton) [reply src/components/CaptureMatching/CaptureImage.js]: That works for me

**Review by nmcharlton — APPROVED** [review]

**Pull request merged**